en/components/grids_templates/export-excel.md

```diff
@@ -48,7 +48,12 @@ The Excel Exporter service can export data to excel from the @@igxName. The data
 
```

Comment:
The original topic states the grid is exported in table format - " The data export functionality is encapsulated in the IgxExcelExporterService class and the data is exported in MS Excel table format.". This needs to be changed for the Hierarchical grid.